Live/Classes/GameTimer.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <chrono>
+typedef std::chrono::system_clock Clock;
+
+class GameTimer {
+public:
+  enum DayPhase { DAY, NIGHT };
+
+  GameTimer();
+  ~GameTimer();
+  static GameTimer *instance();
+
+  int getTime();
+  int getDays();
+
+  DayPhase getCurrentDayPhase();
+
+
+private:
+  static GameTimer *shared;
```

Comment:
Rename to instance.

Reply:
Can't do that. We have a method called instance already.

Reply:
change that to `getInstance`